app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,16 @@
+<div style="box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;">
+<%= link_to category_path(category) do %>
+
+  <div id="<%= dom_id category %>" class="col">
+    <div class="toolbar2">
+      <%= link_to(category_path(category)) do %>
+        <img style="width:10%" src="<%= category.icon %>" />
```

Comment:
The images of the categories are not displayed on the UI. 

- [ ] Let's please also correct this bug in order for the user to have the images of the categories displayed on the UI.